### PR TITLE
Fixed 'get_encodings_from_content' incompatible with Python3.

### LINKED ISSUE #172: utils.get_encodings_from_content doesn't work with Python 3
Since the parameter `content` is byte string in Python 3, the re variable `find_charset`/`find_pragma`/`find_xml` won't match against `content` and complains:
`cannot use a string pattern on a bytes-like object`
It will be better to use explicit byte- prefix for regular expressions.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/40614752-utils-get_encodings_from_content-doesn-t-work-with-python-3?utm_campaign=plugin&utm_content=tracker%2F418367&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F418367&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by adoal**
This patch fixes #172  :)

### CODE REVIEW

File: requests_toolbelt/utils/deprecated.py [hunk 1]
```diff
@@ -4,6 +4,17 @@
 
 from requests import utils
 
+from .._compat import PY3 as _PY3
+
+_re_charset = r'<meta.*?charset=["\']*(.+?)["\'>]'
```
Comment (sigmavirus24) [inline requests_toolbelt/utils/deprecated.py]: You can actually write each of these as

```
br'...'
```

We're always expecting bytes and `r'...' == br'...'` on Python 2 and `r'...'.encode('ascii') == br'...'` on Python 3. =)

File: requests_toolbelt/utils/deprecated.py [hunk 2]
```diff
@@ -20,15 +31,15 @@ def get_encodings_from_content(content):
     :type content: bytes
     """
     find_charset = re.compile(
```
Comment (sigmavirus24) [inline requests_toolbelt/utils/deprecated.py]: Can we move the compile calls up to the module level while we're here? Thanks!

File: requests_toolbelt/utils/deprecated.py [hunk 3]
```diff
@@ -6,15 +6,17 @@
 
 from .._compat import PY3 as _PY3
```
Comment (sigmavirus24) [inline requests_toolbelt/utils/deprecated.py]: Can you remove this too? We don't need it anymore.

**Comment by adoal** [comment]
It's fixed. Thank you for your patience with a newbie :)

File: requests_toolbelt/utils/deprecated.py (hunk 3 above)
Comment (adoal) [inline requests_toolbelt/utils/deprecated.py]: It's fixed. Thank you for your patience with a newbie :)

**Comment by sigmavirus24** [comment]
Thanks for taking the time to fix this @adoal!

**Pull request merged**